images/git/main.tf

```diff
@@ -0,0 +1,81 @@
+terraform {
+  required_providers {
+    apko = {
+      source                = "chainguard-dev/apko"
+      configuration_aliases = [apko.alpine]
+    }
+  }
+}
+
+locals {
+  accounts = toset(["nonroot", "root"])
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "dev" { source = "../../tflib/dev-subvariant" }
+
+module "tagger" {
+  for_each = local.accounts
+  source   = "../../tflib/tagger"
+
+  depends_on = [
+    module.test-latest-alpine,
+    module.test-latest-alpine-dev,
+    module.test-latest-wolfi,
+    module.test-latest-wolfi-dev,
+  ]
+
+  tags = merge(
+    # Tagging format for alpine/musl variant:
+    # - latest
+    # - latest-dev
+    # - latest-root
+    # - latest-root-dev
+    # - $v
+    # - $v-dev
+    # - root-$v
+    # - root-$v-dev
+    (each.key == "nonroot" ?
```

Comment:
I'm not super happy with this approach, since if we ever extend this to >2 this ternary will fall apart. but this was the "cleanest" way I could come up with

Reply:
We've talked about moving accounts and run-as into TF config so it can be shared more easily, this might be worth doing sooner if this makes this simpler.

Reply:
These comments help a lot, thanks for adding them!